convert2rhel/utils.py

```diff
@@ -158,12 +296,9 @@ def run_subprocess(cmd, print_cmd=True, print_output=True):
         if print_output:
             loggerinst.info(line.rstrip("\n"))
 
-    # Call communicate() to wait for the process to terminate so that we can get the return code by poll().
-    # It's just for py2.6, py2.7+/3 doesn't need this.
-    process.communicate()
+    process.wait()
```

Comment:
Do you think that we're safe from the deadlock described in https://docs.python.org/3.6/library/subprocess.html#subprocess.Popen.wait:

> This will deadlock when using stdout=PIPE or stderr=PIPE and the child process generates enough output to a pipe such that it blocks waiting for the OS pipe buffer to accept more data. Use [Popen.communicate()](https://docs.python.org/3.6/library/subprocess.html#subprocess.Popen.communicate) when using pipes to avoid that.

We use `stdout=PIPE`.

Reply:
Hmm, good catch! I think communicate should still be used. I will revert to that.